feature/gribi/otg_tests/mpls_in_udp_scale/README.md

```diff
@@ -1,145 +1,501 @@
-  /qos/interfaces/interface/input/scheduler-policy/schedulers/scheduler/state/conforming-octets:
-  /qos/interfaces/interface/input/scheduler-policy/schedulers/scheduler/state/exceeding-pkts:
-  /qos/interfaces/interface/input/scheduler-policy/schedulers/scheduler/state/exceeding-octets:
-
-  # afts next-hop counters
+  # AFTs Next-Hop state (Verification)
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/index:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/type:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/mpls/state/mpls-label-stack:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/src-ip:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dst-ip:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/src-udp-port:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dst-udp-port:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/ip-ttl:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/udp-v6/state/dscp:
   /network-instances/network-instance/afts/next-hops/next-hop/state/counters/packets-forwarded:
   /network-instances/network-instance/afts/next-hops/next-hop/state/counters/octets-forwarded:
+  /network-instances/network-instance/afts/next-hops/next-hop/state/ip-address: # NH IP
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/index:
+  /interfaces/interface/subinterfaces/subinterface/ipv4/neighbors/neighbor/state/link-layer-address:
+
+  # AFTs Next-Hop-Group state (Verification)
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/next-hop: # Verify NHs in NHG
 
-  # afts state paths set via gRIBI
-  # TODO: https://github.com/openconfig/public/pull/1153
-  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/src-ip:
-  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/dst-ip:
-  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/ip-ttl:
-  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/dst-udp-port:
-  #/network-instances/network-instance/afts/next-hops/next-hop/mpls-in-udp/state/dscp:
+  # AFTs Prefix Entry state (Verification)
+  /network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/next-hop-group:
+  /network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/next-hop-group:
 
 rpcs:
   gnmi:
-    gNMI.Set:
-      union_replace: true
-      replace: true
```

Comment:
Since there is a configuration pushed to the DUT, gNMI.Set is still used in this test.

Reply:
meaning, can you add this text back in?